libs/megaparse/tests/pdf/test_detect_ocr.py

```diff
@@ -1,10 +1,15 @@
+import os
+
+import pytest
 from megaparse.parser.strategy import determine_strategy
 from megaparse_sdk.schema.parser_config import StrategyEnum
 
+list_pdf = os.listdir("./tests/pdf/ocr")
+
 
-def test_strategy_all():
-    pdf = "./tests/pdf/sample_pdf.pdf"
+@pytest.mark.parametrize("pdf", list_pdf)
+def test_strategy_all(pdf):
     strategy = determine_strategy(
-        pdf, threshold_pages_ocr=0.2, threshold_image_page=0.3
+        f"./tests/pdf/ocr/{pdf}", threshold_pages_ocr=0.2, threshold_image_page=0.3
```

Comment:
Can we have also the inverse tests ? Native pdf documents should not be considered scanned